docs/genealignmentsynteny.md

```diff
@@ -0,0 +1,213 @@
+## Work through
+
+Seeing as this can be quite a complicated to set up here's a walk through.
+
+### Step 1 - Set up the directories
+
+Lets set up the system as if we want to run it on a bird genome.
+
+```bash
+
+mkdir -p gene_alignment_prep/scripts/
+
+cp treeval/bin/treeval-dataprep/* gene_alignment_prep/scripts/
+
+mkdir -p gene_alignment_prep/raw_fasta/
+
+mkdir -p gene_alignment_data/bird/csv_data/
+
+mkdir -p synteny/bird/
+```
+
+The naming of the bird folder here is important, keep this in mind.
+
+So now we have this structure:
+
+```
+~/treeval-resources
+    │
+    ├─ synteny/
+    │   └─ bird/
+    │
+    ├─ gene_alignment_data/
+    │   └─ bird/
+    │         └─ csv_data/
+    │
+    └─ gene_alignment_prep/
+        ├─ scripts/
+        └─ raw_fasta/
+```
+
+### Step 2 - Download some data
+
+First, let's download out sytenic alignment data. I think the zebra finch would be good against the Chicken.
+
+```bash
+cd  synteny/bird/
+
+curl https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/003/957/565/GCA_003957565.4_bTaeGut1.4.pri/GCA_003957565.4_bTaeGut1.4.pri_genomic.fna.gz -o bTaeGut1_4.fasta.gz
+
+gunzip bTaeGut1_4.fasta.gz
+```
+
+This leaves us with a file called `bTaeGut1_4.fasta` the genomic assembly of bTaeGut1_4 (the Darwin Tree of Life ID for this species) also known as `taeniopygia guttata`, the Australian Zebra Finch.
+
+Now lets move into the raw_data folder and download some data, this may take some time.
+
+```bash
+cd  ../../gene_alignment_prep/raw_data/
+
+curl https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/016/699/485/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b_cds_from_genomic.fna.gz -o GallusGallus-GRCg7b.cds.fasta.gz
+
+curl https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/016/699/485/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b_genomic.fna.gz -o GallusGallus-GRCg7b.cdna.fasta.gz
+
+curl https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/016/699/485/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b_protein.faa.gz -o GallusGallus-GRCg7b.pep.fasta.gz
+
+curl https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/016/699/485/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b_rna.fna.gz -o GallusGallus-GRCg7b.rna.fasta.gz
+
+```
+
+Now that's all downloaded we need to prep it. At this point it is all still gzipped (the .gz on the end denotes that the file is compressed) in this format we can't use it. So lets use some bash magic.
+
+This is a for loop written in bash, this will look through the current folder we are in for files ending with .fasta.gz and then gunzip them. This unzips the file, uncompresses it so it is usable, and then runs our python script, `GA_data_prep.py`.
```

Comment:
```suggestion
This is a for loop written in bash, this will look through the current folder we are in for files ending with `.fasta.gz` and then gunzip them. This unzips the file, uncompresses it so it is usable, and then runs our python script, `GA_data_prep.py`.
```